languages/rust/exercises/concept/enums/.meta/design.md

```diff
@@ -0,0 +1,44 @@
+# Design
+
+## Goal
+
+The goal of this exercise is to explore the concept of [enums][thebook] in Rust. 
+
+## Learning Objectives 
+
+After completing the exercise, the user should:
+
+- Know of the existence of the `enum` keyword.
+- Know how to define enum variants.
+- Know how to assign values to enum variants.
+
+## Out of scope
+
+- Tuple enums.
+- Generic enums.
+- Memory and performance characteristics.
+
+## Concepts
+
+The Concepts this exercise unlocks are:
+
+- `enums-basic`: know of the existence of the `enum` keyword; know how to define enum variants; know how to assign values to enum variants; 
+
+## Prequisites
+
+This exercise's prerequisites Concepts are:
+
+- `strings-basic`: log lines are `string` values.
+- `numbers-basic`: enum values are assigned a numeric value.
```

Comment:
The fact that an enum discriminates between its variants by assigning each a numeric tag is an implementation detail, and should not be part of the basic introduction to enums.

```suggestion
```